src/esn.calendar.libs/app/components/event/form/event-form.controller.js

```diff
@@ -603,6 +605,15 @@ function CalEventFormController(
     return $scope.attendees.users.concat(attendees, $scope.attendees.resources, resources);
   }
 
+  function updateLocationLink(location) {
+    let updatedLocation = location;
+
+    if (!location.startsWith('http')) {
+      updatedLocation = `//${location}`;
+    }
+    $window.open(updatedLocation, '_blank');
+  }
+
```

Comment:
```suggestion
  function openLocationLink(location) {
    $window.open(location.startsWith('http') ? location : `//${location}`, '_blank', 'noopener');
  }
```

Notice that I have also added the third parameter, which is `'noopener'`. This is very important to prevent [a kind of phishing attack](https://dev.to/ben/the-targetblank-vulnerability-by-example).

Reply:
yes i had notice that